stubs/setuptools/pkg_resources/__init__.pyi

```diff
@@ -1,72 +1,87 @@
-import importlib.abc
 import types
-import zipimport
-from _typeshed import Incomplete
-from abc import ABCMeta
-from collections.abc import Callable, Generator, Iterable, Iterator, Sequence
-from io import BytesIO
+from _typeshed import Incomplete, StrPath, Unused
+from collections.abc import Callable, Generator, Iterable, Iterator, Mapping, Sequence
+from io import BufferedIOBase, BytesIO
+from itertools import chain
+from os import PathLike
+from pkgutil import get_importer as get_importer
 from re import Pattern
-from typing import IO, Any, ClassVar, Literal, Protocol, TypeVar, overload, type_check_only
+from typing import IO, Any, BinaryIO, ClassVar, Final, Literal, NamedTuple, NoReturn, Protocol, TypeVar, overload, type_check_only
 from typing_extensions import Self, TypeAlias
+from zipfile import ZipInfo
+from zipimport import zipimporter
 
 from ._vendored_packaging import requirements as packaging_requirements, version as packaging_version
 
+# using `map[Requirement]` directly causes errors with pytype https://github.com/google/pytype/issues/1324
```

Comment:
Even if pytype allowed it I'm not sure using the `map` type directly would be a good idea.